tests/by-util/test_install.rs

```diff
@@ -566,3 +568,76 @@ fn test_install_copy_then_compare_file_with_extra_mode() {
 
     assert!(after_install_sticky != after_install_sticky_again);
 }
+
+#[test]
+#[cfg(not(windows))]
+fn test_install_and_strip() {
+    const SYMBOL_DUMP_PROGRAM: &str = "objdump";
+
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+
+    let source_file = if cfg!(target_os = "macos") {
+        "helloworld_macos"
+    } else {
+        "helloworld_linux"
+    };
+
+    let target_file = "helloworld_installed";
+
+    scene
+        .ucmd()
+        .arg("-s")
+        .arg(source_file)
+        .arg(target_file)
+        .succeeds()
+        .no_stderr();
+
+    let output = Command::new(SYMBOL_DUMP_PROGRAM)
+        .arg("-t")
+        .arg(at.plus(target_file))
+        .output()
+        .unwrap();
+    let stdout = String::from_utf8(output.stdout).unwrap();
+    assert!(!stdout.contains("main"));
+
+    scene
+        .ucmd()
+        .arg("-s")
+        .arg("--strip-program")
+        .arg("/usr/bin/strip")
+        .arg(source_file)
+        .arg(target_file)
+        .succeeds()
+        .no_stderr();
+
+    let output = Command::new(SYMBOL_DUMP_PROGRAM)
+        .arg("-t")
+        .arg(at.plus(target_file))
+        .output()
+        .unwrap();
+    let stdout = String::from_utf8(output.stdout).unwrap();
+    assert!(!stdout.contains("main"));
+
+    let stderr = scene
```

Comment:
please move that into a separate test
it helps debugging when tests are focusing on one thing